tests/ssl-opt.sh

```diff
@@ -3945,6 +3996,7 @@ run_test    "Session resume using tickets: CAMELLIA-256-CCM" \
             -s "a session has been resumed" \
             -c "a session has been resumed"
 
+requires_cipher_enabled "ARIA" "CCM"
```

Comment:
Typo? Same with `ARIA-192-GCM` at L4014 and `ARIA-256-GCM` at L4029.
```suggestion
requires_cipher_enabled "ARIA" "GCM"
```

Reply:
Right! This one is almost certainly the effect of copy-paste. Thanks for noticing ;)